packages/expo-application/expo-module.config.json

```diff
@@ -1,6 +1,5 @@
 {
-  "name": "expo-application",
```

Comment:
Thanks for removing these names 👍 We had a use case for different package name in the past, but it's useless if it matches the npm package name.